package.json

```diff
@@ -29,7 +29,7 @@
     "read-pkg": "^5.0.0",
     "registry-auth-token": "^4.0.0",
     "semver": "^7.1.2",
-    "tempy": "^0.5.0"
+    "tempy": "^0.7.0"
```

Comment:
latest release is 1.0.0 but okay 🤷🏼

Reply:
renovate typically upgrades to the latest within the current major and separately upgrades majors. we'll probably be getting another PR sometime soon